CONTRIBUTING.md

```diff
@@ -79,6 +81,16 @@ With Python 3:
 
 Now the website is available at `http://localhost:8080`.
 
+## Using Docker
+
+1. Clone this repository and the [backend repository](https://github.com/VLCTechHub/VLCTechHub-api) in the same folder.
```

Comment:
We wouldn't need to clone the API repo, as we'd only need the published Docker image of the API. I'll take a look into it.